src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -250,7 +250,9 @@ public override async Task<bool> Execute()
                 {
                     if (await this.LoadGitHubClient(true))
                     {
-                        return commandEvent.IsSuccessful = await this.GitHubSubmitManifests(manifests);
+                        return commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                            manifests,
+                            $"New package: {manifests.InstallerManifest.PackageIdentifier} version {manifests.InstallerManifest.PackageVersion}");

```

Comment:
Since you already have GetPRTitle, can that function also be called here that way the New command is not a special case. This would also make it easier to ensure consistent titles in case we ever need to modify the PR title again in the future.

Reply:
New command now calls GetPRTitle() to generate PR title